src/defs.h

```diff
@@ -624,11 +635,11 @@ typedef enum { // must be coordinated with s_fds_handle_desc in common.cpp
     RECVFROM = 0,
     RECVFROMMUX,
     SELECT,
-#ifndef WIN32
+#ifndef __windows__
     POLL,
     EPOLL,
-    SOCKETXTREME,
 #endif
+    SOCKETXTREME,
```

Comment:
To be aligned with `s_fds_handle_desc` from `common.cpp`:
```c++
    RECVFROM = 0,
    RECVFROMMUX,
    SELECT,
#ifndef __windows__
    POLL,
    EPOLL,
#ifdef USING_VMA_EXTRA_API
    SOCKETXTREME,
#endif // USING_VMA_EXTRA_API
#endif // __windows__
```

Reply:
yes but then at `sockperf.cpp` it doesn't recognized `SOCKETXTREME` 

https://github.com/Mellanox/sockperf/blob/6e5cea3fa55c6c82d7578333d164a27e72c71cac/src/sockperf.cpp#L3350

Reply:
ok, keep it as is